### PR TITLE
feat: create branch / import repo cards

### PR DESCRIPTION
**Pull request by alexnm**
A bit of a late minute feature, added cards for create branch (XTD-237) / import repo (XTD-200). Also ensured buttons work for the list view on the header (see screenshots)

Create branch - grid
image

Create branch - list
image

Import repo - grid
image

Import repo - list
image

### CODE REVIEW

**Comment by lenasmayor** [comment]
Can we add the `+` icon instead of using the written + in the buttons?

File: packages/app/src/app/pages/Dashboard/Components/Header/index.tsx [hunk 1]
```diff
@@ -96,19 +106,44 @@ export const Header = ({
...
                 width: 'auto',
+                transition: `color ease-in`,
+                transitionDuration: theme => theme.speeds[2],
+                '&:hover': {
+                  color: '#e5e5e5',
+                },
               })}
             >
-              + Import Repo
+              + Create branch
```
Comment (lenasmayor) [inline packages/app/src/app/pages/Dashboard/Components/Header/index.tsx:146]: Can we add the `+` icon instead of using the written +?

**Review by lenasmayor — CHANGES_REQUESTED** [review]

File: packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      onClick={() => openCreateSandboxModal({ initialTab: 'import' })}
+      css={css({
+        display: 'flex',
+        width: '100%',
+        alignItems: 'center',
+        justifyContent: 'center',
+        fontSize: 4,
+        fontFamily: 'inherit',
+        fontWeight: 'normal',
+        color: '#e5e5e5',
```
Comment (lenasmayor) [inline packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx:21]: Can we change icon and font colors to `#999` and font-size to 13px?

File: packages/app/src/app/pages/Dashboard/Components/Branch/NewBranch.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      as="a"
+      href={v2DraftBranchUrl(owner, repoName)}
+      css={css({
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        textDecoration: 'none',
+        fontSize: 3,
+        fontWeight: 'normal',
+        color: '#e5e5e5',
```
Comment (lenasmayor) [inline packages/app/src/app/pages/Dashboard/Components/Branch/NewBranch.tsx:21]: Can we change icon and font colors to #999 and font-size to 13px?

**Review by lenasmayor — CHANGES_REQUESTED** [review]

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6977/commits/4da9b40f014fdc4c634497e38c8ae02944a338b9 is at https://pr6977.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx (hunk 2 above)
Comment (alexnm) [reply packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx:21]: I tried to align the typography/colors to the repo cards on the page. This is how it looks with 13/#999
image

Comment (alexnm) [reply packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx:21]: I can change the color, but I would sync the font size with the other cards in the page

Comment (alexnm) [reply packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx:21]: actually, nevermind, it is in sync with the text under the card (x branches) so it looks ok, will push both changes

**Review by tristandubbeld — APPROVED** [review]

**Review by olarclara — APPROVED** [review]
➕

**Pull request merged**